content/advisories/2021-11-11-transaction-broadcast.zh-tw.md

```diff
@@ -0,0 +1,28 @@
+
++++
+title = "公告：對交易自動重新進行的回應"
+date = "2021-11-11"
+[ author ]
+  name = "Dogecoin Foundation"
++++
+
+幣安今日暫停了狗狗幣的提款功能，並指出他們發現狗狗幣的＂小問題＂。對此我們想做出回應：
+
+數個月前 (請注意，在稍早發文中說的是一年前，但首次確認的提及是在四月) 幣安告知我們他們遇到一些交易卡住的情況，交易無法成功被礦工打包。我們建議幣安對這些交易使用 RBF (Replace-By-Fee) 機制 - 以使用較高手續費的新交易取代原先的交易。值得重視的是，RBF 能使先前的交易失效（因此也可說是＂取代＂），所以這個作法被優先建議。由於那些交易沒有啟用 RBF，我們建議手動建立使用相同 input 的新交易，強制使原先的交易失效。
```

Comment:
Several months ago (noting while an earlier post stated a year, first confirmed mention is April) Binance notified **a group of Dogecoin Core maintainers** that Binance had cases where transactions were stuck, meaning the transactions were not being mined successfully. **The maintainers** suggested Binance use RBF (replace by fee) on these transactions, which would replace the original transactions with a new transaction with a higher fee paid. Notably this was suggested because replace by fee invalidates the previous transaction (hence "replace"). As the transaction had RBF disabled, it was recommended that Binance create a new transaction manually, which would consume the same inputs to forcibly invalidate the previous transaction.

---

Please correct the highlighted sentences as above in the translation, according to the latest updates.